index.html

```diff
@@ -173,6 +173,31 @@ <h2>About <br> Myself</h2>
                     <li class="list-item"><a href="#">Skills</a> <i class="fa fa-angle-right"></i></li>
                 </ul>
             </section>
+            <section class="contact_form">
+              <div class="contact-div">
+                <h2 class="contact-h2">Contact me</h2>
+                <p class="contact-paragraph">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+                </p>
+                <form action="/contact-me" method="post">
```

Comment:
- [ ] Please, consider that action value should be similar to `https://formspree.io/f/{form_id}`. Actually, your form is not submitting - just redirect to the not available `contact-me` page.